cli/src/command.rs

```diff
@@ -68,13 +68,25 @@ impl SubstrateCli for Cli {
 			"rococo-staging" => Box::new(service::chain_spec::rococo_staging_testnet_config()?),
 			"rococo-local" => Box::new(service::chain_spec::rococo_local_testnet_config()?),
 			"rococo" => Box::new(service::chain_spec::rococo_config()?),
-			path if self.run.force_kusama => {
-				Box::new(service::KusamaChainSpec::from_json_file(std::path::PathBuf::from(path))?)
-			},
-			path if self.run.force_westend => {
-				Box::new(service::WestendChainSpec::from_json_file(std::path::PathBuf::from(path))?)
+			path => {
+				let path = std::path::PathBuf::from(path);
+
+				let starts_with = |prefix: &str| {
+					path.file_name().map(|f| f.to_str().map(|s| s.starts_with(&prefix))).flatten().unwrap_or(false)
+				};
+
+				// When `force_*` is given or the file name starts with the name of one of the known chains,
+				// we use the chain spec for the specific chain.
+				if self.run.force_rococo || starts_with("rococo") {
+					Box::new(service::RococoChainSpec::from_json_file(std::path::PathBuf::from(path))?)
```

Comment:
`path` is already a `PathBuf` (line 72)
```suggestion
					Box::new(service::RococoChainSpec::from_json_file(path)?)
```